Assets/Spriter2UnityDX/Editor/AnimationBuilder.cs

```diff
@@ -83,6 +86,36 @@ public class AnimationBuilder : UnityEngine.Object {
 					}
 				}
 			}
+            if (animation.soundlines == null)
+            {
+                //this means there are no sounds
+                //set events to null in case there were sounds on a previous import
+                AnimationUtility.SetAnimationEvents(clip, null);
```

Comment:
On second thought, there's a bit of a problem. The above line will get rid of any user-defined AnimationEvents.